src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java

```diff
@@ -298,6 +299,13 @@ public BaseCurrencyNetwork fromString(String string) {
                 preferences.setIgnoreTradersList(Arrays.asList(StringUtils.deleteWhitespace(newValue)
                         .replace(":9999", "").replace(".onion", "")
                         .split(",")));
+
+        // referralId
+        referralIdInputTextField = addLabelInputTextField(root, ++gridRow, Res.get("setting.preferences.refererId")).second;
```

Comment:
Perhaps this field could get a grayed out text that disappears when focusing saying something like "optional referral code" to make it clear it's just optional and generate less confusion.

Reply:
I think also it would be good to make it clear that it's just optional.

Reply:
Yes, good point. I also though it might require some extra info. I will add a prompt text when not filled.